src/other-scripts/emails/style.scss

```
@@ -2,15 +2,7 @@
 	display: none;
 }
 
-.editor-styles-wrapper .block-editor-block-list__layout {
-	:not(code) {
-		font-family: Arial, Helvetica, sans-serif !important;
-	}
-
-	.wp-block-button {
-		margin-left: 0 !important;
-		margin-right: 0 !important;
-		padding: 0 !important;
-	}
+.editor-styles-wrapper .block-editor-block-list__layout .wp-block-button {
+	margin-left: 0 !important;
+	margin-right: 0 !important;
```

```suggestion
.editor-styles-wrapper .block-editor-block-list__layout {
	:not(code) {
		font-family: Arial, Helvetica, sans-serif !important;
	}

	.wp-block-button {
		margin-left: 0 !important;
		margin-right: 0 !important;
		padding: 0 !important;
	}
```

Small regression -- see https://github.com/Automattic/newspack-plugin/pull/3326